etc/kayobe/pulp-repo-versions.yml

```diff
@@ -44,12 +44,13 @@ stackhpc_pulp_repo_rocky_9_1_baseos_version: 20230228T044432
 stackhpc_pulp_repo_rocky_9_1_crb_version: 20230228T044432
 stackhpc_pulp_repo_rocky_9_1_extras_version: 20230228T044432
 stackhpc_pulp_repo_rocky_9_1_highavailability_version: 20230228T044432
-stackhpc_pulp_repo_rocky_9_2_appstream_version: 20230609T075101
-stackhpc_pulp_repo_rocky_9_2_baseos_version: 20230607T131006
-stackhpc_pulp_repo_rocky_9_2_crb_version: 20230607T131006
-stackhpc_pulp_repo_rocky_9_2_extras_version: 20230607T131006
-stackhpc_pulp_repo_rocky_9_2_highavailability_version: 20230607T131006
+stackhpc_pulp_repo_rocky_9_2_appstream_version: 20230627T185020
+stackhpc_pulp_repo_rocky_9_2_baseos_version: 20230627T185020
+stackhpc_pulp_repo_rocky_9_2_crb_version: 20230627T185020
+stackhpc_pulp_repo_rocky_9_2_extras_version: 20230612T072506
+stackhpc_pulp_repo_rocky_9_2_highavailability_version: 20230628T031859
 stackhpc_pulp_repo_treasuredata_4_version: 20221105T035018
 stackhpc_pulp_repo_ubuntu_cloud_archive_version: 20230607T200208
 stackhpc_pulp_repo_ubuntu_focal_security_version: 20230607T200208
 stackhpc_pulp_repo_ubuntu_focal_version: 20230607T200208
+
```

Comment:
White-space change, but end of file whitespace is more correct: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
OK, tox doesn't like it. I guess we should remove.

Reply:
```suggestion
```